File: 240712/특정 숫자 도달하기/reaching-specific-number.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int sum = 0;
        int cnt = 0;

        int[] arr = new int[10];

        for(int i = 0; i < 10; i++){
            arr[i] = sc.nextInt();
        }

        for(int i = 0; i < 10; i++){
            if(arr[i] < 250){
                cnt++;
                sum += arr[i];
            }
            else break;
        }
        
        System.out.println(sum + " " + String.format("%.1f", sum / (double)cnt));
    }
}

File: 240712/특정 숫자 도달하기/reaching-specific-number.js
const fs = require('fs');
const arr = fs.readFileSync(0).toString().trim().split(" ").map(Number);

let sum = 0;
let cnt = 0;

for(let i = 0; i < 10; i++){
    if(arr[i] < 250){
        cnt++;
        sum += arr[i];
    }
    else{
        break;
    }

}

console.log(sum, Math.round(sum / cnt * 10) /  10);